docs/release-notes/4606-get-experiments-consolidation.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Changes**
+
+-  API: the `GetProjectExperiments` endpoint (`/projects/:id/experiments`) is removed and replaced
```

Comment:
I believe we're using double backticks (``), not single (`) in our docs.

Reply:
Pushed new version with ``